src/signals/incident/containers/KtoContainer/components/KtoForm/index.js

```diff
@@ -212,45 +140,42 @@ const KtoForm = ({
             Een antwoord mogelijk, kies de belangrijkste reden
           </HelpText>
 
-          <RadioButtonList
-            aria-describedby="subtitle-kto"
```

Comment:
zullen we deze nog toevoegen aan de checkbox list?

Reply:
🆗